src/pages/about/ValuesBlock.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+
+interface ValuesBlockProps {
+    content: {
+        title: string;
+        description: string;
+        image: {
+            src: string;
+            alt: string;
+        };
+    }[];
+}
+
+const ValuesBlock = (props: ValuesBlockProps) => (
+    <ValuesBlockDiv>
+        {props.content.map((value, index) => (
+          <ValueDiv>
+          <ValueTitle>{value.title}</ValueTitle>
+          <ValueContent>
+            <Description>{value.description}</Description>
+            <Image src={value.image.src} alt={value.image.alt}></Image>
+          </ValueContent>
+          </ValueDiv>
+        ))}
+    </ValuesBlockDiv>
+);
+
+const ValueDiv = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-bottom:30px;
+`
+
+const ValueContent = styled.div`
+    display: flex;
```

Comment:
lets make an issue to resize this component for smaller screens too 
![Screen Shot 2023-06-22 at 11 41 56 PM](https://github.com/cublueprint/cublueprint-react-website/assets/44822892/5be496ed-386f-4bb7-90fe-b0af92f8a0e8)